dotCMS/src/main/java/com/dotcms/rendering/js/JsResponseStrategyFactory.java

```diff
@@ -104,7 +104,7 @@ private String resultToString(final Object result) {
 
         if (result instanceof Map) {
 
-            final Map map = Map.class.cast(result);
+            final Map map = Map.class.cast(result); // note: we do not know what could it be, so we have to handle as a generic Map.
```

Comment:
Even when you dont know the type you should use Object in the generics
? on its own is just anything that extends object, so you should use the following here

final Map<?, ?> map = (Map<?, ?>) result;

Reply:
looks very ugly buddy but I will do the change heheh